jupyter-js-widgets/src/embed-webpack.js

```diff
@@ -7,13 +7,12 @@
 // Load jquery and jquery-ui
 var $ = require('jquery');
 require('jquery-ui');
-$.getQueryParameters = window.$.getQueryParameters;
```

Comment:
Ithink the real problem here is that the jquery version of the doc theme !== the version specified in the package.json